realm/README.md

```diff
@@ -0,0 +1,31 @@
+To configure wolfSSL, use the following command:
+
+```
+./configure --enable-static --enable-opensslall --enable-enckeys --enable-certgen --enable-context-extra-user-data
+sudo make install
```

Comment:
Please change to use configure then make then sudo make install. Trying to run the make under sudo leads to other issues like with make check.